.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+version: 2
+
+.job_template: &job_template
+  machine:
+    enabled: true
+  steps:
+    - checkout
+    - run:
+        command: |
+          cd "${BASEDIR:-.}/${CIRCLE_JOB}"
+          [[ $IMAGE != *"/"* ]] && IMAGE="diginc/${IMAGE}"
```

Comment:
Should this `IMAGE` point to the Pi-hole namespace?

Reply:
I'm afraid my docker hub token being used will cause this to actually deploy over the current prod ftl-build images (I think I have deploy rights there).  

I'm testing the diginc/ftl-build images now to know if that is safe or not.

Reply:
Okay, I can set up pralor with a hub credential and give that bot user a build team access.